packages/@icon-magic/library/README.md

```diff
@@ -0,0 +1,53 @@
+# @icon-magic/library
```

Comment:
Will be good to mark that this package is currently not in use and is deprecated.